app/dashboard/views.py

```diff
@@ -935,7 +935,7 @@ def profile_keywords_helper(handle):
     """
     profile = profile_helper(handle, True)
 
-    keywords = []
+    keywords = list()
```

Comment:
You shouldn't need to change this and is a commonly considered an antipattern in Python in favor of the shorthand `[]`